frameit/frames_generator/Rakefile

```
@@ -46,9 +72,10 @@ task(:generate_device_frames) do
 
   image_assets = []
   devices = Hash.new { |h, k| h[k] = [] }
-  (Dir["raw/Phones/Apple*/Device/*.png"] + Dir["raw/Computers/Apple*/Device/*.png"] + Dir["raw/Tablets/Apple*/Device/*.png"]).each do |current_raw|
+  # According to mac_editor.rb, Mac screenshots do not need frames - do we need to process the Computers folder?
```

Yeah, having the option to frame computers still seems like a good thing to have (or at least having the frames, to one day write the actual code for that if it is currently not (properly) implemented)

---

Understood, so the comment could be changed to this:
```suggestion
  # Frames from the "computers" folder are currently not used by the mac_editor.rb, but may be used in the future if implemented.
```

---

Either that, or just remove completely.

---

OK, removed completely in the new commit.